node-hub/dynamixel-client/main.py

```diff
@@ -0,0 +1,198 @@
+"""
+Dynamixel Client: This node is used to represent a chain of dynamixel motors. It can be used to read positions,
+velocities, currents, and set goal positions and currents.
+"""
+
+import os
+import time
+import argparse
+import json
+
+import pyarrow as pa
+
+from dora import Node
+
+from common.dynamixel_bus import DynamixelBus, TorqueMode
```

Comment:
So I understand that those dynamixel files are used in multiple places. 

But I think it would make more sense to have standalone nodes.

Reply:
Yeah you're right, I'm moving it inside the node so it's a standalone